hs_core/views/utils.py

```diff
@@ -232,14 +232,22 @@ def validate_json(js):
         raise ValidationError(detail='Invalid JSON')
 
 
-def validate_user_name(user_name):
-    if not User.objects.filter(username=user_name).exists():
-        raise ValidationError(detail='No user found for user name:%s' % user_name)
+def validate_user(user_identifier):
+    if not User.objects.filter(username=user_identifier).exists():
```

Comment:
Can we use the util function ```user_from_id()``` here?

Reply:
That's not a bad idea, but my only resistance to doing that is that method raises an Http404 User Not found.  I don't think a core util should be raising http errors and should be more general, but that's the pattern that currently exists.  To change it would domino some refactoring that I don't have cycles to take on now.